tests/app/views/test_users.py

```diff
@@ -1065,6 +1066,45 @@ def test_returns_404_for_nonexistent_supplier(self):
             "supplier_id '{}' not found".format(non_existent_supplier),
             response.get_data(as_text=True))
 
+    def test_only_buyers_returned_with_only_buyers_arg_set_true(self):
```

Comment:
This test and, especially, the one following it, are super weird in that they are expecting users to be returned that aren't mentioned in the test itself. 

Not sure this is the time to fix this particular issue, but just making a note that it sucks.